bukkit/src/main/java/me/leoko/advancedban/bukkit/BukkitMethods.java

```diff
@@ -80,7 +81,7 @@ public String getFromUrlJson(String url, String key) {
             JSONParser jp = new JSONParser();
             JSONObject json = (JSONObject) jp.parse(new InputStreamReader(request.getInputStream()));
 
-            String[] keys = key.split("\\|");
+            String[] keys = Regex.Split.VERTICAL_LINE.split(key);
```

Comment:
Regex isn't needed here. String#split has a fast-path for single character delimiters, including where the backslash escape character is used.